demos/ripples.html

```diff
@@ -12,7 +12,7 @@
       <svg width="{{width}}" height="{{height}}">
         <rect x="0" y="0" width="{{width}}" height="{{height}}" fill="#1B1B1B" />
 
-        {{#each points.items key="key" as |point|}}
```

Comment:
I've modified `Iterable` to support creating an `ArrayIterator` from any non-array which supports `forEach`. It iterates the `forEach`, putting the values in an array which are then used in an `ArrayIterator`. Please let me know if you can think of a way to generate an iterator which uses the `forEach` directly. 

As this is somewhat generic, I didn't subclass `TestEnvironment`. Let me know if you think this is incorrect